### ISSUE TITLE
Question/Bug: Support more than 6 significant digits

### ISSUE THREAD

**Issue by ckennerknecht** [questdb/c-questdb-client#3]
Hello,

I am having problems (and maybe found a bug) with exporting large double values to my local QuestDB instance.
I have made some test cases to demonstrate them. 
Any help is appreciated!

## General Question:
Is there a way to increase the significant digits for double values that are sent via ILP,
because per default only 6 significant digits are supported.
I could not find any configuration for this.

--> See test cases 1 and 2

## Problem
When the scientific notation is used (7 digits or more before the decimal point) in the line protocol and it seems like QuestDB can not parse this.
[InfluxDB supports the scientific notation.](https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_reference/#examples)

--> See test case 3

## Setup
```sql
CREATE TABLE 'test' (
 ts TIMESTAMP,
 sensor_id SYMBOL,
 value double
)
timestamp(ts) PARTITION BY Day;
```
```cpp
 sender
   .table(table_name)
   .symbol(id_name, "sensor_1"_utf8)
   .column(value_name, value)
   .at_now();  
```

### Case 1
```cpp
  double value = 1.1234567;
```
Result in WebUI:
![image](https://user-images.githubusercontent.com/47369419/170052414-de30c8b4-ef01-4c39-82d5-ef72ecec7380.png)

Rounded to only 6 significant digits.

### Case 2
```cpp
 double value = 654321.123;
```
Result in WebUI:
![image](https://user-images.githubusercontent.com/47369419/170053743-b7f2b3d2-c464-438e-8c75-250c337c8230.png)

Only 6 significant digits. Digits after decimal point are ignored.

### Case 3
```cpp
 double value = 7654321.123;
```
Console:
> Could not flush buffered messages: An established connection was aborted by the software in your host machine.

QuestDB Log:
> E i.q.c.l.t.LineTcpConnectionContext [2756] could not parse measurement, INVALID_FIELD_VALUE at 38, line (may be mangled due to partial parsing): 'test,sensor=sensor_1 value=7.65432e+06'

As said, the scientific notation, that is used for 7 digits or more, can not be parsed by QuestDB.

**Comment by amunra** [comment]
Thank you for the detailed bug report!

This is really two bugs.

1. The first one is a limitation in the float-to-string logic used in the library. I'm looking right now to fix this.
2. The second is a bug in QuestDB's parsing logic for ILP: I'll create a new bug report there and cross-reference this one.

**Comment by amunra** [comment]
Work in progress. See: https://github.com/questdb/c-questdb-client/tree/float_to_string_improvements

**Comment by ckennerknecht** [comment]
Is the second bug fixed by this or have you opened a QuestDB issue for it?

**Comment by amunra** [comment]
I still need to raise a QuestDB bug for it, that said the next client release will not use exponential notation so it should just work across versions.

**Comment by amunra** [comment]
Fix released as library version 1.1.0.

**Comment by amunra** [comment]
The exponential notation issue has been logged as: https://github.com/questdb/questdb/issues/2225.
